Source/Android/jni/MainAndroid.cpp

```diff
@@ -511,6 +513,22 @@ JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_Initialize(J
   UICommon::InitControllers(WindowSystemInfo(WindowSystemType::Android, nullptr, nullptr, nullptr));
 }
 
+JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_InitializeVR(JNIEnv* env,
+                                                                                 jclass,
+                                                                                 jobject activity,
+                                                                                 jstring systemname)
+{
+  if (IsVREnabled())
+  {
+    jobject obj = env->NewGlobalRef(activity);
```

Comment:
When I saw this `NewGlobalRef` call without a corresponding `DeleteGlobalRef` call, I was thinking: Is that a bug or not? I'm assuming it's not a bug, but to prevent others in the future from wondering the same thing, you could make the interface cleaner by calling `NewGlobalRef` inside `InitVROnAndroid` instead of expecting the caller of `InitVROnAndroid` to do it.